android/titanium/src/java/org/appcelerator/titanium/util/TiAnimationBuilder.java

```diff
@@ -329,6 +295,10 @@ private AnimatorSet buildPropertyAnimators(int x, int y, int w, int h, int paren
 			addAnimator(animators, ObjectAnimator.ofFloat(view, "alpha", toOpacity.floatValue()));
 		}
 
+		if (elevation != -1) {
```

Comment:
I think we should avoid doing equality checks with floats. I'd prefer that we change it to the following instead.
```java
if (elevation >= 0) {
	addAnimator(animators, ObjectAnimator.ofFloat(view, "elevation", elevation));
}
```

Also, would you mind updating the `Animation.yml` file with this new property please?
We can add this feature to 9.1.0.

The rest looks great. Thanks!

Reply:
you're right, that looks better! Fixed + docu added